.gitignore

```diff
@@ -2,3 +2,4 @@ bin/
 gopath/
 *.sw[ponm]
 .DS_Store
+stage1.aci
```

Comment:
Wrt bin being for executables, we do use it more generally elsewhere for
compiled artefacts, eg ace validator. For better or worse (arguably build/
or output/ would be better..)

I tend towards that pattern since then I know I can blow away all build
output at any time (and also not clutter the root of the repo).
On Jan 28, 2015 12:02 PM, jonathan.boulle@coreos.com wrote:

> I see what you mean. What about deriving a relative path from rkt, ie
> assuming stage1 is located in the same dir? That makes the tar ball case
> super easy too.
> On Jan 28, 2015 11:30 AM, "Vito Caputo" <notifications@github.com> wrote:
>
>> In .gitignore
>> <https://github.com/coreos/rocket/pull/436#discussion_r23715272>:
>>
>> > @@ -2,3 +2,4 @@ bin/
>> >  gopath/
>> >  *.sw[ponm]
>> >  .DS_Store
>> > +stage1.aci
>>
>> I always run rkt as bin/rkt during development. I'll move it into bin if
>> you think it's more appropriate. The stage1.aci isn't an executable though,
>> which is true for everything else we put in bin. Since I build via ./build,
>> I tend to run bin/rkt which expects stage1.aci where I have it.
>>
>> Maybe the default should be an absolute path derived at build time so I
>> can continue running bin/rkt and others ./rkt so everyone's happy. Is
>> there an equivalent to gcc -DSTAGE1_PATH=/foo/bar.aci for go build?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/coreos/rocket/pull/436/files#r23715272>.
>>
>